scheduling/allocator.rb

```diff
@@ -233,7 +236,7 @@ def self.candidate_hosts(request)
       ds = ds.where(location_id: request.location_filter) unless request.location_filter.empty?
       ds = ds.where(allocation_state: request.allocation_state_filter) unless request.allocation_state_filter.empty?
       ds = ds.where(Sequel[:vm_host][:family] => request.family_filter) unless request.family_filter.empty?
-      ds = ds.exclude(total_cores: 14, total_cpus: 14) unless request.family == "standard-gpu"
+      ds = ds.exclude(total_cores: 14, total_cpus: 14) unless request.gpu_count > 0
```

Comment:
Since we have the `family_filter`, we might remove the total_cores:14 filter. No need to maintain it. GPU vm hosts should have `family: standard-gpu` anyway.

Reply:
I need to to make some changes in the allocator anyways and would address this then.